<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LAKB5DO" failures="1" tests="2" name="APIAutomationTest" time="7.424" errors="0" timestamp="2024-05-04T09:20:40 ICT">
  <testcase classname="APITest" name="test1" time="4.367">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at APITest.test1(APITest.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="APITest" name="test2" time="2.89"/>
</testsuite> <!-- APIAutomationTest -->
